package.json

```diff
@@ -30,7 +30,7 @@
     "build:vite": "ts-node-esm --transpileOnly --project=.vite/tsconfig.json .vite/build.ts",
     "build:types": "concurrently \"tsc -p ./packages/mermaid/tsconfig.json --emitDeclarationOnly\" \"tsc -p ./packages/mermaid-mindmap/tsconfig.json --emitDeclarationOnly\"",
     "build:watch": "pnpm build:vite --watch",
-    "build": "pnpm clean; concurrently \"pnpm build:vite\" \"pnpm build:types\"",
+    "build": "(pnpm clean || true) && concurrently \"pnpm build:vite\" \"pnpm build:types\"",
```

Comment:
|| true is not the right workaround. 

can you try renaming clean to cleanup here and line 29?

Reply:
I've replaced with cleanup and it still doesn't work.

![image](https://user-images.githubusercontent.com/82361490/194521322-401b020e-7497-483e-8122-a8b741c44c1a.png)

![image](https://user-images.githubusercontent.com/82361490/194521262-35327298-29bd-404d-8af7-70f616724fbd.png)

I've also tried with a random name:

![image](https://user-images.githubusercontent.com/82361490/194521730-3b07ee14-981b-4d0e-a183-3cfb0bd7af44.png)

![image](https://user-images.githubusercontent.com/82361490/194521787-492a800a-7f8f-4451-adb5-47b45d73b2b2.png)

But they work if semi-colon is replaced:

![image](https://user-images.githubusercontent.com/82361490/194522404-abbaecb8-86bd-4646-84ca-3e66217148eb.png)

![image](https://user-images.githubusercontent.com/82361490/194522328-166cabd3-5e4d-4a0c-968b-15e4cb1fc69f.png)

Reply:
You can actually remove clean from the root package.json.

Try if `pnpm run -r clean` works inside build?

Reply:
I have removed clean from root, and added the command you gave.

This was the first run:

![image](https://user-images.githubusercontent.com/82361490/194527304-6f672b45-551e-4f20-89ae-b61ad4136889.png)

Then I put a space between clean and ; and this is second run:

![image](https://user-images.githubusercontent.com/82361490/194527206-c2cc13b6-1466-4b19-b8b8-e4cf2c00d762.png)

Here the second part of the script is not running.

Reply:
You can delete `;` and use &&.